modules/ding_ting_frontend/ding_ting_frontend.info

```diff
@@ -20,6 +20,7 @@ dependencies[] = pm_existing_pages
 dependencies[] = strongarm
 dependencies[] = ting
 dependencies[] = ting_covers
+dependencies[] = ting_covers_addi
```

Comment:
Now that we have split ting_covers into a core module with plugins I do not think it makes sense to force a dependency to one of the plugins.

Instead I suggest that we enable it in `ding_ting_frontend_install()`.